sheets-core/src/commonMain/kotlin/com/dokar/sheets/SheetBehaviors.kt

```diff
@@ -85,6 +85,7 @@ open class SheetBehaviors(
     val collapseOnClickOutside: Boolean = true,
     val extendsIntoStatusBar: Boolean = false,
     val extendsIntoNavigationBar: Boolean = false,
+    val isModal: Boolean = true
```

Comment:
Can we get a better name for it? Since `modal` might feel confusing and unrelated to `CoreBottomSheetLayout`. Maybe something like `isOutsideTouchable` or `allowOutsideInteraction`?

Reply:
Totally agree, isModal is a bit confusing.